code/game/machinery/shieldgen.dm

```diff
@@ -553,3 +553,644 @@
 
 #undef ACTIVE_SETUPFIELDS
 #undef ACTIVE_HASFIELDS
+
+//Modular Shield Generator Start
+/obj/machinery/modular_shield_gen
```

Comment:
```suggestion
/obj/machinery/modular_shield_generator
```

there's no reason to abbreviate.

Reply:
you underestimate the amount of times i`ll have to rewrite that guess ill get started

Reply:
done